applications/kalpana/app.json

```diff
@@ -0,0 +1,139 @@
+    "description": "",
+    "dynamicExecSystem": false,
+    "execSystemConstraints": null,
+    "execSystemId": "frontera",
+    "execSystemExecDir": "${JobWorkingDir}",
+    "execSystemInputDir": "${JobWorkingDir}",
+    "execSystemOutputDir": "${JobWorkingDir}/output",
+    "execSystemLogicalQueue": "small",
+    "archiveSystemId": "cloud.data",
+    "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+    "archiveOnAppError": true,
+    "isMpi": false,
+    "mpiCmd": null,
+    "cmdPrefix": null,
+    "parameterSet": {
+      "appArgs": [],
+      "containerArgs": [],
+      "schedulerOptions": [
+        {
+          "name": "TACC Scheduler Profile",
+          "description": "Scheduler profile for HPC clusters at TACC",
+          "inputMode": "FIXED",
+          "arg": "--tapis-profile tacc-apptainer",
+          "notes": {
+            "isHidden": true
+          }
+        },
+        {
+          "name": "Slurm job name",
+          "description": "Set the slurm job name to be identical to the Tapis job name.",
+          "inputMode": "FIXED",
+          "arg": "--job-name ${JobName}",
+          "notes": {
+            "isHidden": true
+          }
+        }
+      ],
+      "envVariables": [
+        {
+          "key": "filetype",
+          "value": "",
+          "description": "Please enter maxele.63.nc for maximum water levels, or maxwvel.63.nc for maximum wind speeds.",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "File type"
+          }
+        },
+        {
+          "key": "polytype",
+          "value": "polygon",
+          "description": "Please enter your choice of vector shape.",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "Vector shape",
+            "enum_values": [
+              {
+                "polyline": "polyline"
+              },
+              {
+                "polygon": "polygon"
+              }
+            ]
+          }
+        },
+        {
+          "key": "contour",
+          "value": "contourrange",
+          "description": "Please enter your choice of contour type.",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "Contour type",
+            "enum_values": [
+              {
+                "contourrange": "contourrange"
+              },
+              {
+                "contourlevel": "contourlevel"
+              }
+            ]
+          }
+        },
+        {
+          "key": "range",
+          "value": "",
+          "description": "Please enter contour information; for contourrange use format 'min max interval' (e.g. '0 5 0.5') and for contourlevel provide custom set of contourlevels (e.g. '0 1 2 3 4 5 6 7 8 9 10 11 12').",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "Contour Information"
+          }
+        }
+      ],
+      "archiveFilter": {
+        "includes": ["workingDirectory"],
```

Comment:
Is the output gong to this input dir? If so, we may want to move it somewhere else. Best not to archive the input dir I think